test/Models/mobilenet-without-batchnorm.mlir

```diff
@@ -1381,14 +1381,14 @@ func.func @mobilenet(%arg0: tensor<1x224x224x3xf32>) -> tensor<1x1001xf32> {
     linalg.yield %1307 : f32
   } -> tensor<1x1x1x1001xf32>
   //
-  // CHECK: %[[subview:.*]] = memref.subview
-  // CHECK: %[[subview1:.*]] = memref.subview
-  // CHECK: %[[subview2:.*]] = memref.subview
+  // C_HECK: %[[subview:.*]] = memref.subview
```

Comment:
Honestly, I think we can get rid of these CHECKs entirely. Just leave the xsmm call and it should be enough for these entire models.

Reply:
Got rid of these specific CHECKs, but did not touch the rest.